examples/create_rosa_cluster/create_rosa_sts_cluster/byo_oidc_cluster/README.md

```diff
@@ -0,0 +1,26 @@
+# ROSA STS cluster creation example
+
+This example shows how to create an STS _ROSA_ cluster. _ROSA_ stands for Red Hat Openshift Service on AWS
+and is a cluster that is created in the AWS cloud infrastructure.
+
+To run it:
+
+* Provide OCM Authentication Token 
+
+  OCM authentication token that you can get [here](https://console.redhat.com/openshift/token).
+    ```
+    export TF_VAR_token=...
+    ```
+
+* Provide OCM environment by setting a value to url    
+    ```
+    export TF_VAR_url=...
+    ```
+
+* Decide STS operator_role_prefix    
+    ```
+    export TF_VAR_operator_role_prefix=...
+    ```
+
+and then run the `terraform apply` command.
+
```

Comment:
Please add information on how to generate/ create your own OIDC, and what the values of `oidc_endpoint_url`, `oidc_private_key_secret_arn`, should be

Reply:
Those will come in a separate PR after I deal with STS module to include this possibility

Reply:
Got it now